### ISSUE TITLE
access contol to the OGC schema?

### ISSUE THREAD

**Issue by michcfr** [FraunhoferIOSB/FROST-Server#1974]
Hello,
Has someone implemented access contol, like the RBAC or hierarchical RBAC, to the OGC schema?
Regards

**Comment by hylkevds** [comment]
Yes, FROST-Server can do highly flexible, fine-grained access control that can be tuned to your specific requirements. An example plugin that does this can be found here: https://github.com/hylkevds/FROST-Server.Plugin.Projects/
This plugin uses Basic Auth, with an in-database user table by default, but the user and role management can also be done through KeyCloak.

Since access control is not something one wants to accidentally get wrong, we do suggest contacting us for support in setting it up.

**Comment by michcfr** [comment]
This is exactly what I am looking for. Thank you :-)
I'll proceed to its installation and brb to you.

By the way, where can I get the list of available FROST plugins?

**Comment by hylkevds** [comment]
There is a list for the official plugins on the Plugin Settings page: https://fraunhoferiosb.github.io/FROST-Server/settings/plugins.html
The Projects plugin will move there too for the 2.4.0 release.
